packages/server/test/integration/cypress_spec.js

```diff
@@ -846,7 +846,10 @@ describe('lib/cypress', () => {
     // for headed projects!
     // also make sure we test the rest of the integration functionality
     // for headed errors! <-- not unit tests, but integration tests!
-    it('logs error and exits when project folder has read permissions only and cannot write cypress.config.js', function () {
+    // this test is skipped because its failure causes websocket integration tests to fail.
+    // this test should be revisited, as the error it's asserting on probably can never be
+    // actually thrown by Cypress.
```

Comment:
This test is never run in CI (where tests are run as root), and it always fails locally.

The `ERROR_WRITING_FILE` error is thrown by `_logWriteErr` in `packages/server/lib/util/settings.ts`, which is referenced by `_write()` in the same file. `_write()` is only referenced by `writeForTesting()` in the same file, which is _only referenced in tests_, and not runtime code.